pkg/metadata/key.go

```diff
@@ -0,0 +1,55 @@
+package metadata
+
+import (
+	"strings"
+
+	"github.com/hanfei1991/microcosm/pkg/errors"
+	"github.com/pingcap/ticdc/dm/pkg/log"
+	"go.uber.org/zap"
+)
+
+const (
+	// MetaStoreKeyBase is the common prefix of keys in dataflow system
```

Comment:
I have added a `pkg/adapter` for the key prefix management. It is ported by DM. What do you think if we should resue that or just manage them here?